arrow-libs/fx/arrow-fx-coroutines/src/jvmTest/kotlin/arrow/fx/coroutines/RaceNJvmTest.kt

```diff
@@ -119,10 +119,12 @@ class RaceNJvmTest : StringSpec({
       }
     }
 
+    /* This seems to not be true anymore
```

Comment:
🤯, really!?
I guess we want to stick to what `select` does 🤔

Reply:
It's been flaky in the last few versions...

Reply:
Strange, do you now with what assertion error?